src/portable/renesas/link/hcd_link.c

```diff
@@ -156,28 +106,6 @@ typedef struct
 //--------------------------------------------------------------------+
 static hcd_data_t _hcd;
 
-static uint32_t disable_interrupt(void)
```

Comment:
do you still remember why this disable_interrupt()/enable_interrupt() is dropped ? Or is it not needed in the first place ?

Reply:
This has moved to the link_rx.h file, as it is specific to the RX family